### ISSUE TITLE
Support hostnames as value for ManagerHost

### ISSUE THREAD

**Issue by raballew** [engelmi/hirte#276]
## Please describe what you would like to see

Support hostnames as values for `ManagerHost` of the `hirte-agent` configuration.

```
[hirte-agent]
NodeName=laptop
ManagerHost=foo
ManagerPort=2020
```

```
[hirte-agent]
NodeName=laptop
ManagerHost=bar.localdomain
ManagerPort=2020
```

## Please describe your use case

I was trying to setup a multi node env using virtual machines with various different subnet configurations (different IP address ranges) and had to use a different configuration for every subnet even though the hostnames are identical.

**Comment by pypingou** [comment]
I'm thinking to remove this one from our roadmap. If someone wants to work on it I'm fine with it but I don't think it's a priority for us atm

**Comment by engelmi** [comment]
This has been implemented in #340. Closing.
